easybuild/easyconfigs/c/Conda/perl-app-cpanminus-1.7039.eb

```diff
@@ -0,0 +1,41 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2015 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_Conda'
+
+name = "perl-app-cpanminus"
+version = "1.7039"
+variant = "Linux-x86_64"
+
+homepage = "https://github.com/bioconda/bioconda-recipes"
+description = """ Conda is a platform- and
+    language-independent package manager that sports
+    easy distribution, installation and version
+    management of software. The bioconda channel is a
+    Conda channel providing bioinformatics related
+    packages. This repository hosts the corresponding
+    recipes. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+requirements = "perl-app-cpanminus=1.7039"
```

Comment:
use `requirements = "%(name)s-%(version)s"`

that way, you make sure that `--try-software-version` will work correctly on this easyconfig, and that you won't forget to update the version in two places in case you update this manually